app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -104,8 +107,48 @@ class MapActivity : ComponentActivity() {
                     MapView()
                     BuildingInfoUIComponent()
                     Hud(
-                        PlayerGlobalData(false, 420),
-                        listOf(PlayerGlobalData(false, 32), PlayerGlobalData(false, 56)),
+                        Player(
+                            user = User(
+                                id = 4572,
+                                name = "User test 1",
+                                bio = "",
+                                skin = Skin.default(),
+                                stats = Stats(0,0,0,0,0),
+                                trophiesWon = listOf(),
+                                trophiesDisplay = mutableListOf(),
+                            ),
+                            balance = 420,
+                            ownedLocations = listOf(),
+                            roundLost = null,
+                        ),
+                        listOf(
+                            Player(
+                                user = User(
+                                    id = 4573,
+                                    name = "User test 2",
+                                    bio = "",
+                                    skin = Skin.default(),
+                                    stats = Stats(0,0,0,0,0),
+                                    trophiesWon = listOf(),
+                                    trophiesDisplay = mutableListOf(),
+                                ),
+                                balance = 32,
+                                ownedLocations = listOf(),
+                                roundLost = null,
+                            ), Player(
+                                user = User(
+                                    id = 4574,
+                                    name = "User test 3",
+                                    bio = "",
+                                    skin = Skin.default(),
+                                    stats = Stats(0,0,0,0,0),
+                                    trophiesWon = listOf(),
+                                    trophiesDisplay = mutableListOf(),
+                                ),
+                                balance = 56,
+                                ownedLocations = listOf(),
+                                roundLost = null,
+                            )),
```

Comment:
What is all of that, is it used for testing?

Reply:
Will be fixed by next week's Great Refactoring of MapActivity! no need to bother Tamara about it !

Reply:
I only changed this part to make it compile